### ISSUE TITLE
Can't load base binarys via proxy since update to 5.

### ISSUE THREAD

**Issue by johannes85** [vercel/pkg#1163]
I use pkg in a docker containe.
To access the internet I use a company proxy which worked fine with the 4.x release of pkg.

Since updating to 5.x (no other changes) the download isn't workin anymore:
```
> Fetching base Node.js binaries to PKG_CACHE_PATH
> Not found in remote cache:
  {"tag":"v3.0","name":"node-v10.24.1-linux-x64"}
```

The environment variables http_proxy and https_proxy are set.

Downloading the same file via wget is working.

After going back to 4 with ```npm install -g pkg@4``` it is working again.

**Comment by robertsLando** [comment]
show the command you are using to package the application

**Comment by johannes85** [comment]
```
npm install -g pkg@4
npm install
pkg --out-path /dist -t node10-linux-x64 server.js
```

or

```
npm install -g pkg@4
npm install
http_proxy=....... https_proxy=....... pkg --out-path /dist -t node10-linux-x64 server.js
```

**Comment by robertsLando** [comment]
Strange, seems that binary is present in releases: `https://github.com/vercel/pkg-fetch/releases/download/v3.0/node-v10.24.1-linux-x64`

**Comment by robertsLando** [comment]
try to run pkg with `--debug` flag if you can see more info, maybe the fetch fails for some reason

**Comment by robertsLando** [comment]
Fixed on pkg-fetch 3.1 wait for next pkg release
